databricks/koalas/namespace.py

```diff
@@ -86,7 +87,9 @@
 ]
 
 
-def from_pandas(pobj: Union["pd.DataFrame", "pd.Series"]) -> Union["Series", "DataFrame"]:
+def from_pandas(
+    pobj: Union["pd.DataFrame", "pd.Series", "pd.Index"]
+) -> Union["Series", "DataFrame", "Index"]:
     """Create a Koalas DataFrame or Series from a pandas DataFrame or Series.
```

Comment:
How about also add a description about Index to the docstring ? 
Something like `Create a Koalas DataFrame, Series or an Index  from a pandas DataFrame, Series or an Index.`